test/e2e/common/fixtures.py

```diff
@@ -51,6 +59,7 @@ def xgboost_churn_endpoint(sagemaker_client):
         replacements=replacements,
     )
     assert model_resource is not None
+    sleep_until_k8s_resource_arn(model_resource)
```

Comment:
remove sleep_until_k8s_resource_arn method and please make a PR to accept wait_period and period length in [load_and_create_resource](https://github.com/aws-controllers-k8s/test-infra/blob/53f590e6a5831ec32824d41de2ca88e9b5c22135/src/acktest/k8s/resource.py#L94) method of acktest package

wait is already part of [wait_resource_consumed_by_controller](https://github.com/aws-controllers-k8s/test-infra/blob/53f590e6a5831ec32824d41de2ca88e9b5c22135/src/acktest/k8s/resource.py#L213) method which can be re-used

there shouldn't be any additional wait needed for arn since it's returned onCreate and wait_resource_consumed_by_controller method is checking if `status` != nil. If status is not nil, arn should be there

Reply:
Am rewriting the PR now